src/Modal/Modal.tsx

```diff
@@ -285,11 +285,10 @@ export const Modal: React.FC<Props> = ({
             </div>
             <div
               css={css(
-                title &&
-                  description && {
-                    marginTop:
-                      size === "large" ? 24 : size === "medium" ? 16 : 12,
-                  },
+                (title || description) && {
```

Comment:
Im not quite sure about having description here? Does the description render in the top section?

Reply:
I think the logic is right but the title is possibly confused?

this makes sense `(title || description) ? defaultMargin : noMargin`

(gosh this component is getting tangly though, really want to chop it up)

Reply:
> Does the description render in the top section


it looks like it does
![image](https://user-images.githubusercontent.com/743976/130693545-d7bb8dfd-4986-4fc5-9745-eb2d085a6db0.png)

Reply:
yeah, the description does render in the top section - very tangly component FOR SURE